spec/models/miq_dialog_spec.rb

```diff
@@ -0,0 +1,28 @@
+describe MiqDialog do
+  context '.seed' do
+    it 'seeds from the core with correct metadata' do
+      root = Rails.root.join('product', 'dialogs', 'miq_dialogs')
+      allow(described_class).to receive(:find_by)
```

Comment:
I think `expect(described_class).to receive(:find_by).once.with(...)` would be much simpler than `allow` and `expect().to have_received()`.